### PR TITLE
Added codeowners file

### PR DESCRIPTION
**Pull request by richclement**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Default owners for everything in the repo unless a later match takes precedence
+*       @sonatype/nxrm-techleads
```
Comment (Blacktiger) [inline .github/CODEOWNERS:2]: weird spacing, also a missing newline at the end of the file

**Review by Blacktiger — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (richclement) [reply .github/CODEOWNERS:2]: Why the newline? Github's sample files do not contain a newline at the end of the file.

Comment (Blacktiger) [reply .github/CODEOWNERS:2]: We put newlines at the end of all our files as a matter of style, it probably doesn't really matter

Comment (Blacktiger) [reply .github/CODEOWNERS:2]: presumably we use that as a style because of https://stackoverflow.com/a/729795

Comment (richclement) [reply .github/CODEOWNERS]: Thanks for the explanation. Fixed

**Pull request merged**